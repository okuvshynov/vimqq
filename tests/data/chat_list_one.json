{"n_chat_queries": 2, "n_stream_queries": 1, "n_deltas": 3, "n_non_stream_queries": 1}
