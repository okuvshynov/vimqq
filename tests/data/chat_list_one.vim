function! WriteAndQuit(t)
    :normal q
    execute "write chat_list_one.out"
    execute "qa!"
endfunction

:put!=range(1,5)
:normal ggV5j
:execute "normal! \<Esc>"
:'<,'>QQ -s @mock hello
call timer_start(1000, "WriteAndQuit")
