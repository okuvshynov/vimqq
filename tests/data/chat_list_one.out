00:00>l=166
